portfolio/src/main/webapp/script.js

```diff
@@ -67,3 +72,32 @@ function getMax() {
 function deleteComment() {
   fetch("/delete-list", {method: 'POST'})
 }
+
+async function isLoggedIn() {
+  const response = await fetch('/login-status',);
+  const status = await response.json();
+  return status;
+}
+
+google.charts.load('current', {'packages':['corechart']});
```

Comment:
I believe you are including this same `script.js` file in both your `chart.html` but also your existing HTML files.  This means that when the other webpages load, they are also going to be running this same code that loads the charts API (which will probably result in errors, since they aren't also loading `https://www.gstatic.com/charts/loader.js` and they won't have a `chart-container` element.

I think you could fix this by either:
1) having a separate JavaScript file with this code, maybe called something like `chart.js`
or
2) creating a function to load the charts, and only calling that function when chart.html loads.  Something like:
```
function loadChartsApi() {
  google.charts.load('current', {'packages':['corechart']});
  google.charts.setOnLoadCallback(drawChart);
}
```

Reply:
I chose to create a new function as opposed to a new file because I felt it would keep the codebase cleaner and better conform to the current design style. I now have a function that is called only when chart.html loads. This will address any errors caused by pages running unnecessary code.